eng/Version.Details.xml

```diff
@@ -6,6 +6,8 @@
     <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="6.0.0-beta.20573.2">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>35bddd4fbfab8da3518fb920250d7c9e0c3138ff</Sha>
+      <!--<SourceBuild RepoName="arcade" ManagedOnly="true"/> -->
+
```

Comment:
This needs an update to version of `Microsoft.DotNet.Arcade.Sdk` package so it can work; the current version is too old and can't be found.

    error NU1102: Unable to find package Microsoft.SourceBuild.Intermediate.arcade with version (= 6.0.0-beta.20573.2) 

Probably https://github.com/dotnet/command-line-api/pull/1172?